spinnman/transceiver.py

```diff
@@ -663,6 +663,8 @@ def _update_machine(self):
         # Get the coordinates of the boot chip
         version_info = self.get_scamp_version()
 
+        time.sleep(5)
```

Comment:
Why _5_ seconds? Is this only needed with toroids? Would the big machine need longer?

Reply:
This is now literally the only change here, and I don't think it is needed.  I will therefore close this pull request.  Reopen if desired...